src/pages/office/components/d255/d255.ts

```diff
@@ -34,6 +48,18 @@ export class D255Component implements OnChanges {
     }
   }
 
+  getD255OrDefault(): string | null {
+    if (this.d255 === null) {
```

Comment:
you can get rid of the `=== null`, just use `!this.d255` in the if statement

Reply:
removed === null